packages/js/internal-js-tests/babel.config.js

```diff
@@ -1,7 +1,3 @@
 module.exports = {
-	presets: [
-		'@babel/preset-typescript',
-		'@wordpress/babel-preset-default',
-	],
+	presets: [ '@babel/preset-typescript', '@wordpress/babel-preset-default' ],
```

Comment:
nitpick, but obviously these files have not been prettier formatted in the past, it'd be more ideal not to wholesale include formatting changes in a PR since there isn't actually a relevant change.

Reply:
Interestingly, this change was introduced by commit merging the trunk to this branch. I'll merge it again and see if it fades away.

Reply:
It's no longer there after https://github.com/woocommerce/woocommerce/pull/34468/commits/a5534717418ec312c419b87955bb6c1609ad25d3